subprojects/DirectX-Headers.wrap

```diff
@@ -0,0 +1,5 @@
+[wrap-git]
+directory = DirectX-Headers-1.0
+
+url = https://github.com/microsoft/DirectX-Headers.git
+revision = v1.606.4

```

Comment:
On what exactly do we rely from this project? Can you, please, itemize? I am trying to understand how often we might need to bump up this revision.

Reply:
We depend on the header and external symbols of:

https://github.com/microsoft/DirectX-Headers/blob/main/include/directx/dxcore_interface.h
https://github.com/microsoft/DirectX-Headers/blob/main/src/dxguids.cpp

We shouldn't have to bump the version unless we modify the libva-utils code with usage of new interfaces, which can be bumped with any new change.

Is the concern to depend specifically on v1.606.4 or to depend on a fixed version number?

Reply:
> Is the concern to depend specifically on v1.606.4 or to depend on a fixed version number?

Fixed version number. I don't have any insight whether v1.606.4 is the minimum required. 

But if you start using subprojects you must have some fixed version here I think rather than rely on a tip of the main branch. It's just that I don't like hardcodes, but I've no alternative. So, it'll go as you suggested.

Reply:
I lowered the dependency version. Most platforms from the previous message all have packages for different versions starting from v1.0.2